tests/variables.rs

```diff
@@ -0,0 +1,76 @@
+#[cfg(test)]
+mod tests {
+    use yaiwr::Calc;
+
+    #[test]
+    fn variables_declaration() {
+        struct TestCase<'a> {
+            prog: &'a str,
+            key: &'a str,
+            expected_value: u64,
+        }
+        let mut c = Calc::new();
+        let table = vec![
```

Comment:
It'd be better to have each of these a separate test func. That way you get better test feedback, as at the moment these tests will all succeed or all fail.

Reply:
I have mixed feelings about table testing, but I like them for specific use cases, like the case here - when it's the same test with slightly different variations. It's just convenient to add new cases that way.
In terms of feedback on test failure/success it can be identified by its index/input

Reply:
If it's weird for it to be in the Rustiomatic test suite I can definitely change it!

Reply:
re-wrote as separate test functions :point_right: 1245fa7bec951051415514e70ae41bab9996558d